File: Module-CP/Assignments/assignment14/Pass_by_value/q4.c
#include<stdio.h>
typedef struct HR{
    int id;
    char name[20];
    int salary;
    float commision;

}HR;
HR storeHR();
void display(HR);
int main(){
    HR h1,h2;

    h1=storeHR();
    h2=storeHR();
    
    printf("Details for HR-1:");
    display(h1);
    printf("Details for HR-2:");
    display(h2);
}
HR storeHR(){
    HR hr;

    printf("Enter ID:");
    scanf("%d",&hr.id);
    printf("Enter Name:");
    scanf("%s",hr.name);
    printf("Enter Salary:");
    scanf("%d",&hr.salary);
    printf("Enter Commision:");
    scanf("%f",&hr.commision);

    return hr;

}
void display(HR hr){
    printf("\nID:%d",hr.id);
    printf("\nName:%s",hr.name);
    printf("\nSalary:%d",hr.salary);
    printf("\nCommision:%f",hr.commision);
}

File: Module-Core Java/Assignments/Java Assignments/Assignment 1/ByUsing_Constructors/q4.java
class Employee{
    String name;
    int id;
    double salary;

     Employee() {
        this.name="Pratik";
        this.id=2;
        this.salary=45000;
    }

     Employee(String name, int id, double salary) {
        this.name = name;
        this.id = id;
        this.salary = salary;
    }
    
    void display(){
        System.out.println("Name: "+this.name);
        System.out.println("ID: "+this.id);
        System.out.println("Salary: "+this.salary);
    }
}
public class q4{
    public static void main(String[] args) {
        Employee e1=new Employee();
        Employee e2=new Employee("Snehal",3,50000);

        e1.display();
        e2.display();
    }
}